src/chrome/lib/destroy.js

```diff
@@ -1,15 +1,14 @@
 // Script injected into the page to trigger removal of any existing instances
 // of the Hypothesis client.
-(function () {
```

Comment:
Removing this IIFE means that `annotatorLink` now gets exposed on `window`, which I'm pretty sure is not what we want.